services/QuillLMS/app/views/layouts/pdf.html.erb

```diff
@@ -3,8 +3,10 @@
   <head>
     <title>Quill</title>
     <meta content='text/html; charset=UTF-8' http-equiv='Content-Type'/>
+    <link rel="stylesheet" href=<%= "#{ENV['DEFAULT_URL']}/assets/admin_usage_snapshot_report_pdf.scss" %>>
```

Comment:
I couldn't get this working as a relative path -- I don't think this should be an issue, but if it is I can mess around more or go back to using a `<style>` tag.

Reply:
I remember reading something about relative paths here:  https://github.com/Studiosity/grover#relative-paths

I wonder if that is the issue.